README.md

```diff
@@ -29,7 +29,9 @@ integrations with other projects such as
     version of nvim, v.0.5.x.
 - Ensure [Coursier](https://get-coursier.io/docs/cli-installation) is installed
     on your machine. `nvim-metals` uses Coursier to download and update Metals.
-- Remove `F` from `shortmess`. `set shortmess-=F` _NOTE_: Without doing this,
+- Remove `F` from `shortmess`. `set shortmess-=F`
+    (for lua `vim.o.shortmess = string.gsub(vim.o.shortmess, "F", "") .. "c"`)
```

Comment:
Let's actually use the new syntax for this:

```lua
vim.opt_global.shortmess:remove("F"):append("c")
```

In the future I just might remove all of the non-lua examples.

Reply:
I saw it in `minimal setup` file, but was not sure which is better as I am not strong yet it lua.
Done

Reply:
👍🏼 yea that was from before the introduced the new syntax. Thanks for fixing it!